### PR TITLE
ci: switch to github actions

### PR DESCRIPTION
**Pull request by ahjohannessen**
Given the state of Travis CI OSS I think moving to GH Actions is a good move

### CODE REVIEW

**Comment by ahjohannessen** [comment]
@rossabaker WDYT?

**Comment by ahjohannessen** [comment]
I think that `series/0.x` should be added to `githubWorkflowTargetBranches`

File: build.sbt [hunk 1]
```diff
@@ -1,11 +1,24 @@
 import Dependencies._
 
+lazy val Scala213 = "2.13.3"
+lazy val Scala212 = "2.12.11"
```
Comment (rossabaker) [inline build.sbt:4]: There's a 2.12.12 now

**Review by rossabaker — APPROVED** [review]
This is worth bringing back to series/0.x, if that's going to see any serious maintenance while CE3 cooks. But :+1:

**Pull request merged**